docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -11290,6 +11295,99 @@ ID
 --- 1 row(s) selected.
 ```
 
+<<<
+[[uuid_function]]
+== UUID Function
+Returns a Universal Unique Identifier (UUID) generated according to RFC 4122. 
+A UUID is designed as a number that is globally unique in space and time. 
+Two calls to UUID() are expected to generate two different values, even if 
+these calls are performed on two separate devices not connected to each other.
+
+NOTE: But UUID() can only be used in SELECT list, or very simple expressions to get a UUID.
```

Comment:
Do we prevent it from being used in the WHERE clause? If not, suggested word-smith: "NOTE: UUID() behavior is non-deterministic. Therefore it is best to use it only in SELECT lists. Using it in a WHERE clause, for example, may result in non-deterministic predicate evaluation."

Reply:
We don't prevent it from being used in the WHERE clause. It can be used in some special case. It just we don't know where optimizer will choose to evaluate this function. If in ROOT, it will be one value for per statement, if it pushdown to SCAN, it will be one value for each row.
So I will update the words as you suggested.

Reply:
OK, thanks.